test/NightlyBuildDependency.props

```diff
@@ -0,0 +1,14 @@
+<Project>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML" Version="1.5.0-preview-28305-3" />
```

Comment:
What's the plan for updating these versions? I see only 2 package versions here, should we define a property for them and then use those properties here instead of hard-coding the versions?

Reply:
These versions are only place holder, when running pipeline, the NugetPackageVersionUpdater will update all these out of date version number

---
In reply to: [343777962](https://github.com/dotnet/machinelearning/pull/4444#discussion_r343777962) [](ancestors = 343777962)

Reply:
Would it make sense to instead use a placeholder? Rather than a hard-coded version? I think it deserves a comment explaining what happens so that people understand that we update it on the fly.

Reply:
Yes, I think this is good suggestion, I will sue placeholder to reduce confusion. 

---
In reply to: [343900725](https://github.com/dotnet/machinelearning/pull/4444#discussion_r343900725) [](ancestors = 343900725)